packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js

```diff
@@ -264,14 +289,20 @@ class InvestmentTiers extends React.Component<Props, State> {
         )}
         <FieldArray
           name="investmentTiers.tiers"
-          render={({ push }) => (
+          render={({ push, replace }) => (
             <Field
               name="investmentTiers.newTier"
               ticker={ticker}
               component={AddTierModal}
-              title={`Add the Investment Tier #${value.tiers.length + 1}`}
+              title={
```

Comment:
Could we move this render logic inside the AddTierModal? (Which should now be renamed to `TierModal` or `TierFormModal` or something similar since it also edits now

Reply:
Resolved with latest commit